e2e-execute.sh

```diff
@@ -1,16 +1,24 @@
 #!/usr/bin/env bash
 set -e
 
+shadowJar() {
+  echo "Running shadowJar..."
+  ./gradlew shadowJar
+}
```

Comment:
We can live without the shadowJar being ran first, but I think it ensures that the shadowJar gets built thus avoiding compile issues.

Reply:
Good idea to have this here.  I would only add a `clean` to the front.

Reply:
We are on the same wavelength, I just added that.